src/lib/provable/gadgets/arithmetic.ts

```diff
@@ -49,5 +54,5 @@ function divMod32(n: Field, quotientBits = 32) {
 }
 
 function addMod32(x: Field, y: Field) {
-  return divMod32(x.add(y), 1).remainder;
+  return divMod32(x.add(y), 64).remainder;
```

Comment:
this seems unnecessary, why are we doing this?

Reply:
oh I just saw that quotient bits is now this minus 32. so it should be 33 to match the previous version

Reply:
Why should the quotient bits actually be 1?

Reply:
because the addition of two 32 bit numbers yields a 33 bit number